rickshaw-index

```diff
@@ -1059,6 +1110,20 @@ if (exists $result{'iterations'}) {
                             queue_es_doc("sample", $run_dir . "/" . $this_samp_dir, $iter_idx, $sample_idx);
                         } #opendir samp
                     } #samp pass
+
+                    if ($create_samp_persist_ids_file == 1 or $update_samp_persist_ids_file > 0) {
+                        if ($create_samp_persist_ids_file == 1) {
+                            debug_log(sprintf "Creating sample persistent IDs file %s\n", $samp_persist_ids_file);
+                        }
+                        debug_log(sprintf "Added %d persistent IDs to %s\n", $update_samp_persist_ids_file, $samp_persist_ids_file);
+                        my $file_rc = put_json_file($samp_persist_ids_file, $samp_persist_ids_ref, $sample_persistent_ids_schema_file);
+                        if ($file_rc > 0) {
+                            printf "Could not save the sample persistent IDs file %s\n", $samp_persist_ids_file;
+                            exit 1;
+                        }
```

Comment:
@atheurer This failure condition seems to be tripping on every single CI job but it's working just fine when I test it manually.  Any thoughts?

Reply:
Umm, nevermind...